CHANGELOG.md

```diff
@@ -2,6 +2,11 @@ Main
 ------
 - Fix disputed tag on boundaries with labels only on OSM ways [#475]
 
+Styles 5.4.0
+------
+* add `generate_style` to CLI bin path
+* `generate_style` can take a `.ts` or `.js` ESM Flavor, or a `.json` Flavor
+
```

Comment:
I added a "main" section in the last changelog. Maybe you want to move it down?

Reply:
Bumped the tiles version to 4.11.6

(It already published daily builds with the change as 4.11.5, oh well... any ideas on how we could automate this without delaying the daily publish?)

Reply:
Maybe add a CI check: "If any file was changed in the java src or resources directory, the Basemap.java must also have changed" to check that version is incremented